core/modules/rproc/InfileMerger.cc

```diff
@@ -206,21 +192,24 @@ void InfileMerger::_setQueryIdStr(std::string const& qIdStr) {
 }
 
 
-bool InfileMerger::merge(std::shared_ptr<proto::WorkerResponse> response) {
+bool InfileMerger::merge(std::shared_ptr<proto::WorkerResponse> response, bool last) {
     if (!response) {
         return false;
     }
     // TODO: Check session id (once session id mgmt is implemented)
+    int jobId = response->result.jobid();
```

Comment:
For the sake of code safety, anything that gets initialized once and never changes later, has to be declared like:
```
int const obId = ...
```